dist/profile/templates/buildmaster/lockbox.groovy.erb

```diff
@@ -45,12 +45,20 @@ if (instance.securityRealm == SecurityRealm.NO_AUTHENTICATION) {
   instance.securityRealm = realm
 }
 
-println "Checking authorization strategy"
 /* Every time we run, it's worth re-defining our authorization strategy to make
  * sure that it is correct
  */
 AuthorizationStrategy strategy = new GlobalMatrixAuthorizationStrategy()
-strategy.add(Jenkins.READ, 'anonymous')
+
+[
+    Jenkins.READ,
+    Item.DISCOVER,
+    Item.READ,
+].each { permission ->
+    strategy.add(permission, 'anonymous')
+    strategy.add(permission, 'authenticated')
```

Comment:
This shouldn't actually be necessary – if it is, I would consider it a bug.

Reply:
Yep, should really not be necessary: https://github.com/jenkinsci/jenkins/blob/master/core/src/main/java/hudson/security/SidACL.java#L88...L93